### PR TITLE
KAFKA-18259: Documentation for consumer auto.offset.reset contains in…

### PR DESCRIPTION
**Pull request by frankvicky**
JIRA: KAFKA-18259

Currently, there are some rendering issues with the `auto.offset.reset` config, this PR aims to fix them.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by frankvicky** [comment]
Preview:

Screenshot from 2024-12-16 20-27-31

**Comment by frankvicky** [comment]
Hi @AndrewJSchofield 
Could you please take a look when you have some time?
Many thanks 🙇🏼

File: clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java
Comment (AndrewJSchofield) [inline clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java]: Strictly speaking, you're missing an `</li>` at the end of this line.

**Review by AndrewJSchofield — CHANGES_REQUESTED** [review]
One tiny nit. I confirm that the HTML output is good now.

File: clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java
Comment (frankvicky) [reply clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java]: Yes, you're right 😄 
Just add it.

**Comment by AndrewJSchofield** [comment]
I neglected to approve the changes before I merged. Simply finger-trouble. Approved and merged after a green build.

**Pull request merged**
